sequence-classification/jupiter_shared/results_reader.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os.path
+
+
+class ResultsReader:
+    def __init__(self, base_dir="results", file_prefix=""):
+        self.file_prefix = file_prefix
+        self.base_dir = base_dir
+        if not os.path.isdir(self.base_dir):
+            raise Exception("directory " + self.base_dir + " not found")
+
+    def read_results(self, classifiers):
+        matrices = []
+        for c in classifiers:
+            series_number = 0
+            classifier_name = c[0].name
+            file_name = os.path.join(self.base_dir, self.file_prefix + classifier_name + str(series_number) + ".csv")
```

Comment:
Zmieńmy pętlę `while` na `for` z użyciem funkcji `glob`: [dokumentacja](https://docs.python.org/3/library/glob.html). Wyszukuje ona pliki tak jak bash. Możemy się wtedy pozbyć zmiennej `series_number`.
```Python
from glob import glob

pathname = os.path.join(self.base_dir, self.file_prefix + name + "*.csv")
for filename in glob(pathname):
```